src/main/addons/autoav.ts

```diff
@@ -105,6 +107,9 @@ export default class AutoAV {
     private startRecording(matchInfo: FMSMatchStatus) {
         VmixService.Instance.StartRecording()
             .then(() => {
+                invoke(
```

Comment:
this has a low likelyhood of failing, since it doesn't expect a response, but should probably put this at the end to avoid erroring and messing up the log and match detail storage on L113-L115